pyproject.toml

```diff
@@ -0,0 +1,192 @@
+[build-system]
+requires = ["flit_core >=3.4,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = 'aiotinydb'
+version = '1.2.2'
+description = 'asyncio compatibility shim for tinydb'
+readme = "README.md"
+requires-python = ">=3.6"
+license = { file = "LICENSE.md" }
+authors = [{ name = "Pavel Pletenev", email = "cpp.create@gmail.com" }]
```

Comment:
@d-k-bo Can you please add other contributors and yourself (of course) to this list?

Reply:
As far as I can tell after reading the corresponding paragraph of [PEP 621](https://peps.python.org/pep-0621/#authors-maintainers) and taking a look at other projects (e. g. [TinyDB](https://github.com/msiemens/tinydb/blob/master/pyproject.toml), [rich](https://github.com/Textualize/rich/blob/master/pyproject.toml), [dateutils](https://github.com/dateutil/dateutil/blob/master/setup.cfg)) it is more common to only mention the original author and the current maintainer in the project's metadata. Additional contributors can be listed e. g. in AUTHORS.md.

Reply:
Can you please then add AUTHORS.md?